src/main/kotlin/dev/binclub/paperbin/transformers/AntiNetherRoof.kt

```diff
@@ -179,14 +182,14 @@ object AntiNetherRoof: PaperBinFeature {
 		val x = location.x.toInt()
 		val y = location.y.toInt()
 		val z = location.z.toInt()
-		
+
 		if (x >= worldBorder || z >= worldBorder || x <= negWorldBorder || z <= negWorldBorder) {
 			return false
 		}
 		
-		when (location.world.environment) {
+		when (location.world.environment!!) {
 			World.Environment.NETHER -> {
-				if (y <= 0 || y >= 127) {
+				if (y <= 0 || y >= PaperBinConfig.antiNetherRoofLevel) {

```

Comment:
Directly using the config variable will reduce performance as a config lookup will have to be performed every time the event is called. 

This may be desirable if you want this variable to be changeable at runtime with the config reload command, however it may be more desirable to retrieve the config value once and store it in a field.